src/components/apps/launch/AppLaunchFormSkeleton.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @author psarando
+ *
+ * A Loading Mask for the App Launch Form.
+ */
+import React from "react";
+
+import styles from "./styles";
+
+import {
+    BottomNavigation,
+    BottomNavigationAction,
+    Container,
+    Step,
+    Stepper,
+    StepButton,
+    TextField,
+    Typography,
+    makeStyles,
+} from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+
+const useStyles = makeStyles(styles);
+
+export const StepperSkeleton = () => (
+    <Skeleton width="100%">
```

Comment:
Minor, but a static ID somewhere on the loading skeleton is usually helpful for testing